FirebaseRemoteConfigSwift/Tests/SwiftAPI/FirebaseRemoteConfigSwift_APIBuildTests.swift

```diff
@@ -0,0 +1,233 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+
+import FirebaseCore
+import FirebaseRemoteConfig
+import FirebaseRemoteConfigSwift
+
+final class FirebaseRemoteConfigSwift_APIBuildTests: XCTestCase {
```

Comment:
While I haven't done this in the past, I subclassed `XCTestCase` which will generate a testing log for this class. I figured this makes it easier to assert that the file was built.

Reply:
Have you considered renaming `usage()` to `testUsage()`? (not sure if executing the code would provide any additional useful logs -- just mentioning it as an idea)

Reply:
The issue with executing the code is it then requires a Firebase App to configure so I kept it as a build time check.